plugins/command-line/prism-command-line.js

```diff
@@ -4,6 +4,16 @@
 		return;
 	}
 
+	// Polyfill for pre-ES6 Support, i.e. Internet Explorer
+	if (!String.prototype.endsWith) {
+		String.prototype.endsWith = function (search, this_len) {
+			if (this_len === undefined || this_len > this.length) {
+				this_len = this.length;
+			}
+			return this.substring(this_len - search.length, this_len) === search;
+		};
+	}
```

Comment:
Not a good idea. People will not expect this random plugin to polyfill one specific string function. This plugin having the side effect of polyfilling *some* functions is very surprising, this is why libraries generally don't the polyfilling themselves.

Instead, just make a little function `endsWith` tries to use the built-in `endsWith` and falls back to a simple implementation if not available. Kinda like the `startsWith` function a few lines below here.